src/client/tile.cpp

```diff
@@ -1175,7 +1175,18 @@ bool TextureSource::generateImagePart(std::string part_of_name,
 					core::rect<s32>(pos_from, dim),
 					video::SColor(255,255,255,255),
 					NULL);*/
-			blit_with_alpha(image, baseimg, pos_from, pos_to, dim);
+
+			v2u32 dim_dst = baseimg->getDimension();
```

Comment:
core::dimension2d<u32> would be more consistent with the previous code than v2u32, IMHO.